test/widgets/content_test.dart

```diff
@@ -161,4 +161,33 @@ void main() {
       check(tester.takeException()).isA<AssertionError>();
     });
   });
+
+  group("CodeBlock", () {
+    Future<void> prepareContent(WidgetTester tester, String html) async {
+      await tester.pumpWidget(MaterialApp(home: BlockContentList(nodes: parseContent(html).nodes)));
+      await tester.pump();
+      await tester.pump();
+    }
+
+    testWidgets('with syntax highlighting', (WidgetTester tester) async {
+      // "```dart\nclass A {}\n```"
+      await prepareContent(tester,
+        '<div class="codehilite" data-code-language="Dart"><pre>'
+          '<span></span><code><span class="kd">class</span><span class="w"> </span>'
+          '<span class="nc">A</span><span class="w"> </span><span class="p">{}</span>'
+          '\n</code></pre></div>'
+      );
+
+      expect(find.text('class A {}'), findsOneWidget);
+    });
+
+    testWidgets('without syntax highlighting', (WidgetTester tester) async {
```

Comment:
nit: put "without" case before "with", because it's simpler

Reply:
ack